packages/shared/src/utils/utils.ts

```diff
@@ -121,19 +121,19 @@ export class Utils {
     }
   }
 
-  public static printCombatNumber(num: number): string {
+  public static printCombatNumber(num: number, isCritical = false, criticalMarker = '*'): string {
```

Comment:
the marker should probably be an implementation detail of this function instead of something configurable by callsites. it'll be good to have consistency throughout the app.